src/components/CheckListItem/CheckListItem.tsx

```diff
@@ -3,26 +3,23 @@ import { GrafanaTheme2 } from '@grafana/data';
 import { Checkbox, HorizontalGroup, useStyles2 } from '@grafana/ui';
 import { css, cx } from '@emotion/css';
 
-import { Check, CheckListViewType, CheckType, FilteredCheck, Label } from 'types';
+import { Check, CheckListViewType, CheckType, Label } from 'types';
 import { checkType as getCheckType } from 'utils';
-import { SuccessRateTypes } from 'contexts/SuccessRateContext';
 import { useUsageCalc } from 'hooks/useUsageCalc';
-import { SuccessRateGauge } from 'components/SuccessRateGauge';
+import { LatencyGauge, SuccessRateGaugeCheckReachability, SuccessRateGaugeCheckUptime } from 'components/Gauges';
 
 import { CheckCardLabel } from '../CheckCardLabel';
-import { LatencyGauge } from '../LatencyGauge';
 import { CheckItemActionButtons } from './CheckItemActionButtons';
 import { CheckListItemDetails } from './CheckListItemDetails';
 import { CheckStatusType } from './CheckStatusType';
 
 interface Props {
-  check: FilteredCheck;
+  check: Check;
```

Comment:
In another PR I want to fix the type for `Check`. It's _so_ confusing that the ID is optional and I am assume this is because we are lacking a `DraftCheck` type for check editors?

I feel like we had / have a bandaid of FilteredCheck is the real Check whilst Check is proxying as the non-existent DraftCheck?